debugging/book-library/script.js

```diff
@@ -37,8 +35,8 @@ function submit() {
     alert("Please fill all fields!");
     return false;
   } else {
-    let book = new Book(title.value, title.value, pages.value, check.checked);
-    library.push(book);
+    let book = new Book(title.value, author.value, pages.value, check.checked);
```

Comment:
In the if statement on line 29 there is one more check missing. Please, take a look.

Reply:
Yes, I added a check for the author field. 
Thank you, @tyzia for reviewing my code.